akka-discovery/src/main/scala/akka/discovery/dns/DnsServiceDiscovery.scala

```diff
@@ -135,6 +146,11 @@ private[akka] class DnsServiceDiscovery(system: ExtendedActorSystem) extends Ser
     }
   }
 
+  private val convertToTimeout: PartialFunction[Throwable, Future[Resolved]] = {
+    case _: AskTimeoutException =>
+      Future.failed(new DiscoveryTimeoutException("Dns resolve did not respond within resolveTimeout"))
```

Comment:
can we include the timeout value in the err msg?

Reply:
:+1: